.travis.yml

```diff
@@ -79,7 +79,13 @@ install:
     fi
 
 script:
-  - if [ -n "$BUILD" ]; then cmake . -DMETEREXEC_ROOTACCESS=OFF -DENABLE_OCR=ON -DSML_HOME=/usr/local/src/libsml/sml && make && make test; fi
+  - ls
+  - |
+    if [ -n "$BUILD" ]; then
+      mkdir -p build
+      cd build
+      cmake ../vzlogger -DMETEREXEC_ROOTACCESS=OFF -DENABLE_OCR=ON -DSML_HOME=/usr/local/src/libsml/sml && make && make test;
```

Comment:
Why do we only cd && cmake if the build folder doesn't exist? Why not simply always mkdir -p and get rid of the if?

Reply:
`mkdir -p build && cd build`

Reply:
the if checks whether the env variable BUILD is set. not whether the dir exists.